client/src/components/FormSignUp.jsx

```diff
@@ -1,6 +1,42 @@
 import React from 'react';
 
 function FormSignUp() {
+  const [email, setEmail] = useState("");
```

Comment:
I'm trying to connect the signup form to the front end.  The logic hasn't yet finished

Reply:
I have added more input fields to match the database requriements.